.travis/build.sh

```diff
@@ -7,7 +7,7 @@ if [ ${JAVA_MAJOR_VERSION} -gt 1 ] ; then
   export JAVA_VERSION=${JAVA_MAJOR_VERSION}
 fi
 
-if [ ${JAVA_MAJOR_VERSION} -eq 11 ] ; then
+if [ ${JAVA_MAJOR_VERSION} -eq 11 ] && [ ${TRAVIS_CPU_ARCH} = "amd64" ] ; then
```

Comment:
```suggestion
if [ ${JAVA_MAJOR_VERSION} -eq 11 ] && [ ${TRAVIS_CPU_ARCH} == "amd64" ] ; then
```
Aren't you missing an `==`?

Reply:
@samuel-hawker  Tbh we are using the single "=" even in other operators related scripts.

Reply:
Fair enough, I think it works most of the time there are just weird edge cases where I think double equals is needed.